routes/message.go

```diff
@@ -483,7 +484,7 @@ func (fes *APIServer) SendMessageStateless(ww http.ResponseWriter, req *http.Req
 	// Try and create the message for the user.
 	tstamp := uint64(time.Now().UnixNano())
 	txn, totalInput, changeAmount, fees, err := fes.blockchain.CreatePrivateMessageTxn(
-		senderPkBytes, recipientPkBytes, requestData.MessageText,
+		senderPkBytes, recipientPkBytes, requestData.EncryptedMessageText,
```

Comment:
To maintain backwards compatibility we'll need an extra param here. I think one param for `MessageText` and one for `EncryptedMessageText` works fine.